docs/root/intro/version_history.rst

```diff
@@ -66,6 +67,9 @@ Version history
   that allows ignoring new hosts for the purpose of load balancing calculations until they have
   been health checked for the first time.
 * upstream: added runtime error checking to prevent setting dns type to STRICT_DNS or LOGICAL_DNS when custom resolver name is specified.
+* upstream: added load balancer factory method on cluster, cluster implementation can now control the type of load balancer to be used.
```

Comment:
I think there is a merge issue here. Delete.

Reply:
done